src/fmu/dataio/providers/objectdata/_base.py

```diff
@@ -214,22 +202,12 @@ def _get_named_stratigraphy(self) -> NamedStratigraphy:
         stratigraphy = self.dataio.config.get("stratigraphy", {})
 
         if name not in stratigraphy:
-            return NamedStratigraphy(name=name)
-
-        named_stratigraphy = stratigraphy.get(name)
-        rv = NamedStratigraphy(
-            name=named_stratigraphy.get("name", name),
-            alias=named_stratigraphy.get("alias", []),
-            stratigraphic=named_stratigraphy.get("stratigraphic", False),
-            stratigraphic_alias=named_stratigraphy.get("stratigraphic_alias", []),
-            offset=named_stratigraphy.get("offset", 0.0),
-            top=named_stratigraphy.get("top"),
-            base=named_stratigraphy.get("base"),
-        )
-        if name not in rv.alias:
-            rv.alias.append(name)
+            return StratigraphyElement(name=name)
+
+        if name not in stratigraphy[name]["alias"]:
```

Comment:
I think it's possible to have a `KeyError` here by directly accessing two nests down, but I am not sure. Is alias always there? Not sure what is best, but using `.get()` on these dict elements might be a good idea either way

Reply:
Good point 🙂 I added a `default_factory=list` for the `alias` field, so it should be there in most cases... the corner case being if any user have `alias: None` in their config that will probably not trigger the creation of an empty list.